### DISCUSSION TITLE
Mentioning a user in a comment / text

### DISCUSSION THREAD

**Discussion by parijke** [filamentphp/filament#5209] [Ideas]
It would be an awesome feature if we could mention someone in a text like @xyz and that would trigger a notification for that user with a link to the panel where the mentioning was registered.

I don't know if this is already possible?

**Comment by danharrin** [comment]
I dont think that would be a feature of Filament, you could do it in a model observer though, with database notifications!

**Comment by parijke** [comment]
Maybe you're right. Would be cool though. I am not that comfortable with Laravel to do such thing on my own yet..
